### ISSUE TITLE
Definition of the AA Metric

### ISSUE THREAD

**Issue by Ww-Lee** [zhou13/neurvps#20]
Hello Zhou, I downloaded the pre-trained models from your [Google Drive](https://drive.google.com/drive/folders/1srniSE2JD6ptAwc_QRnpl7uQnB5jLNIZ), and **directly** use **TMM17/checkpoint_latest.pth** and function **AA()** in **neurvps/eval.py** to evaluate TMM17 (Natural Scene) test images. Howerver, the AA curve I generated is different  from the one in paper.  Specifically, I get results **0.127 | 0.291 | 0.503 | 0.743 | 0.855 | 0.921** corresponding to different threshold **[0.5, 1, 2, 5, 10, 20]**.
So I guess there is a problem with the calculation of cumulative histogram in function **AA()**, and then modified it according to my own understanding, and produced pretty similar results.

_(np.argwhere(sorted_err <= threshold)[-1][0] + 1)/ len(loader)_

**Comment by zhou13** [comment]
Why do you think `0.127 | 0.291 | 0.503 | 0.743 | 0.855 | 0.921`  does not match the paper?  This is a screenshot of the result in our paper:
![image](https://user-images.githubusercontent.com/576382/136089939-67b390c4-8ab5-4b35-b784-fd046f159eae.png)

**Comment by zhou13** [comment]
As far as the curve goes, you need to refer to Figure 6. If you want to reproduce Figure 8, you need to use the evaluation code from https://github.com/zihan-z/vpdet_tmm17.

**Comment by zhou13** [comment]
This is because the AA metric is not the curve itself, but the area under the curve of the *angle accuracy curves*, more like CDF rather than percentile.

**Comment by zhou13** [comment]
I will close this issue. Feel free to reopen if you have other findings.
